bridge/bridge.go

```diff
@@ -91,6 +91,11 @@ type DisappearingPortal interface {
 	ScheduleDisappearing()
 }
 
+type PowerLevelPortal interface {
```

Comment:
Hmm, the existing ones are slightly inconsistent, but I think it should have Handling in the name

```suggestion
type PowerLevelHandlingPortal interface {
```